src/features/CopyButton.tsx

```diff
@@ -0,0 +1,27 @@
+import {CopyButton as _CopyButton, Tooltip, ActionIcon} from "@mantine/core"
+import {FiCheck, FiCopy} from "react-icons/fi"
+
+export const CopyButton = (
+    {value} : {value: string}
+  ) => {
+    return (
+      <_CopyButton value={value} timeout={1000}>
+        {({copied, copy}) => (
+          <Tooltip
```

Comment:
Didn't know this existed 🤯 first time using Mantine with you, and it has proven to be a wonderful component lib honestly

Reply:
Credit goes to @marilari88